api-ai-git.py

```diff
@@ -88,12 +90,14 @@ def load_state(commit_hash):
             print("({})  {}  {}".format(i, commit_obj.hexsha, commit_obj.message))
         try:
             num_pressed = int(input("Press number corresponding to which commit you'd like to load the state from: "))
-            if 0 <= num_pressed <= min(len(commits) - 1, 9):
+            # this if statement is some real wild shit fam
+            num_recent_commits = min(len(commits) - 1, 9)
```

Comment:
I actually just changed this purely to get some diffs to show up. Although it should be it's own `variable` anyways. This definitely doesn't relate to issue #1